### PR TITLE
BaSP-MR-47: Admin get put delete post

### PR DESCRIPTION
**Pull request by WalterArielPavicich**
The table contains all the information about the Admins. In each row, I have created a button to delete and edit this information. For the edit button, a request is made to the server that returns the information of the selected admin, in addition, it will open the form with the preloaded data of the Admin to update, being able to edit the field that is necessary. This form contains 3 buttons. The cross button and the cancel button, which closes the form when pressed.  And the button to send the form, that when pressed, a request is made to the server to edit the information and modify the table immediately.
On the other hand, for the Delete button, when pressed, it makes a request to the server and deletes the information, modifying the table immediately.
Finally, under the table I have created a button to add an admin. Pressing it opens the same form mentioned above, which has the empty fields to be filled in. When pressing the add button, it makes a request to the server and modifies the table immediately with the new admin. On the other hand, pressing the cross or the cancel button closes the form.
Each request to the server is shown in a modal with a message depending if the action could be completed or not.

### CODE REVIEW

**Review by MauroCaffesse — APPROVED** [review]

**Comment by MauroCaffesse** [comment]
For me, it's approved!

File: src/Components/Admins/Form/form.module.css [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  justify-content: center;
+  align-items: center;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  overflow: auto;
+  background-color: rgb(0,0,0);
```
Comment (JuanCruzMoreira) [inline src/Components/Admins/Form/form.module.css]: Please keep consistency in color language (use the hex or rgb)

File: src/Components/Admins/Modal/modal.module.css [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  justify-content: center;
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100vw;
+  height: 100vh;
+  overflow: auto;
+  background-color: rgb(0,0,0);
+  background-color: rgba(0,0,0,0.7);
```
Comment (JuanCruzMoreira) [inline src/Components/Admins/Modal/modal.module.css]: Color consistency

File: src/Components/Admins/Table/table.module.css [hunk 3]
```diff
@@ -0,0 +1,42 @@
+/* th {
```
Comment (JuanCruzMoreira) [inline src/Components/Admins/Table/table.module.css]: Delete commented lines

File: src/Components/Admins/index.js [hunk 4]
```diff
@@ -1,11 +1,153 @@
 import styles from './admins.module.css';
+import { useEffect, useState } from 'react';
```
Comment (JuanCruzMoreira) [inline src/Components/Admins/index.js]: import 'React' too.
`import React, { useEffect, useState } from 'react';`

File: src/Components/Admins/index.js [hunk 5]
```diff
@@ -1,11 +1,153 @@
...
+      const response = await fetch(`${process.env.REACT_APP_API_URL}/api/admins`, {
+        method: 'POST',
+        headers: {
+          'Content-type': 'application/json'
+        },
+        body: JSON.stringify(adminToAdd)
+      });
+      if (response.ok) {
+        const res = await response.json();
+        console.log(res);
```
Comment (JuanCruzMoreira) [inline src/Components/Admins/index.js]: delete this console.log() please

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

**Review by JuanCruzMoreira — APPROVED** [review]

**Pull request merged**